map/src/i18n/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Strings } from './iface';
+import en from './langs/en';
+import ru from './langs/ru';
+
+const STRINGS = {
+  en,
+  ru,
+};
+
+type Language = keyof typeof STRINGS;
+
+const selectedLanguage = localStorage.getItem('selectedLanguage');
+
+const queryLanguage = window.location.href.split('/')[3];
+
+const browserLanguage = window.navigator.language.split('-')[0];
```

Comment:
written like this, it means that the language will only be fetched / calculated once, upon browser load. A page refresh shouldn't be required to change the language.

Reply:
Also, this will only check the first language of the navigator, and not check to see if there's perhaps a secondary language that would be better. we should check all languages and filter those we don't know